src/webgpu/api/validation/createBindGroupLayout.spec.ts

```diff
@@ -378,3 +380,30 @@ g.test('storage_texture,layout_dimension')
       });
     }, !success);
   });
+
+g.test('storage_texture,formats')
+  .desc(
+    `
+  Test that a validation error is generated if the format doesn't support the storage usage.
```

Comment:
There is one optional feature which enables storage for a new format. We haven't established a good pattern for testing cases like this in the CTS yet though, so for now it can be a TODO.

```suggestion
  Test that a validation error is generated if the format doesn't support the storage usage.

  TODO: Test "bgra8unorm" with the "bgra8unorm-storage" feature.
```